### DISCUSSION TITLE
Something wrong with the letters

### DISCUSSION THREAD

**Discussion by Tadeoelbeo** [nzp-team/nzportable#818] [General]
I got two mods one is waw sounds and the other is a BO2 mod when i load up the game on psp the letters are all gibberish can someone help me out?

**Comment by Revnova** [comment]
seems like an issue with the mods. psp uses a different resolution than other platforms

**Reply by MotoLegacy** [reply]
not inherently true. same target resolution/graphics are used for the hud. but it probably scales them differently. yes it is a problem with the mods used, but not for that reason.

**Reply by Revnova** [reply]
ah. sorry

**Comment by acronmd** [comment]
yeah i dont think the text in my BO2 mod is PSP compatible (i didnt test it), you should just omit the charset when installing it

**Reply by Tadeoelbeo** [reply]
Yo i firgured it out and your right i just deleted the charset and it worked thanks!
